adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/CoapContext.java

```diff
@@ -45,6 +45,11 @@ public final class CoapContext extends MapBasedTelemetryExecutionContext {
      * The query parameter which is used to indicate an empty notification.
      */
     public static final String PARAM_EMPTY_CONTENT = "empty";
+    /**
+     * The query parameter which is used to indicate, that a piggypacked response is supported by the device.
+     * (Legacy support for device with firmware versions not supporting  piggypacked response.)
+     */
+    public static final String PARAM_PIGGYBACKED = "piggy";
```

Comment:
I can adapt this to `private` as well. Currently it's used in the unit tests, but I could copy it there, if it should be converted into `private`.

Reply:
then maybe use package private scope?